dbt-snowflake/src/dbt/adapters/snowflake/constants.py

```diff
@@ -7,20 +7,20 @@
 DEFAULT_PYTHON_VERSION_FOR_PYTHON_MODELS = "3.9"
 
 
-NATIVE_TABLE_FORMAT = "DEFAULT"
+INFO_SCHEMA_TABLE_FORMAT = "DEFAULT"
```

Comment:
I left the value itself as `DEFAULT` since that was the original default table format. This has been used in other adapters as well, so it's consistent.